### PR TITLE
Update Carbon Native Mobile links (no fathom, please add)

### PR DESCRIPTION
**Pull request by mariuszmickiewicz**
**Changed**

- Updates Carbon Native Mobile links (**no fathom, please add**)

### CODE REVIEW

**Comment by mariuszmickiewicz** [comment]
Locally page renders ok, but in vercel's preview next section's heading shows as a pure text
### Components and pattern

**Review by emyarod — APPROVED** [review]

**Review by andreancardona — APPROVED** [review]
@mariuszmickiewicz looks good to me!

File: src/pages/whats-happening/work-in-progress/index.mdx [hunk 1]
```diff
@@ -36,8 +36,18 @@ provide feedback and have a say in Carbon's direction.
 <CardGroup>
```
Comment (jillianhowarth) [inline src/pages/whats-happening/work-in-progress/index.mdx:36]:
```suggestion
**React Native Sketch kits for mobile**

<CardGroup>
```

File: src/pages/whats-happening/work-in-progress/index.mdx [hunk 2]
```diff
@@ -36,8 +36,18 @@ provide feedback and have a say in Carbon's direction.
...
   <MiniCard
     onClick={() => fathom('trackGoal', 'GC8C8OMU', 0)}
-    title="Carbon mobile sketch kit"
-    href="https://www.sketch.com/s/a3343128-adb6-489c-9e62-709d89ba76e9"
+    title="Carbon Native Mobile (Light theme)"
```
Comment (jillianhowarth) [inline src/pages/whats-happening/work-in-progress/index.mdx:39]:
```suggestion
    title="Light theme"
```

File: src/pages/whats-happening/work-in-progress/index.mdx [hunk 3]
```diff
@@ -36,8 +36,18 @@ provide feedback and have a say in Carbon's direction.
...
+    href="sketch://add-library/cloud/a3343128-adb6-489c-9e62-709d89ba76e9"
+    actionIcon="launch"
+  />
+  <MiniCard
+    title="Carbon Native Mobile (Dark theme)"
```
Comment (jillianhowarth) [inline src/pages/whats-happening/work-in-progress/index.mdx:44]:
```suggestion
    title="Dark theme"
```

File: src/pages/whats-happening/work-in-progress/index.mdx [hunk 4]
```diff
@@ -36,8 +36,18 @@ provide feedback and have a say in Carbon's direction.
...
+    href="sketch://add-library/cloud/1f59f590-6915-47b0-bf06-6fd66209b3b3"
+    actionIcon="launch"
+  />
+  <MiniCard
+    title="IBM Grid template (Native Mobile)"
```
Comment (jillianhowarth) [inline src/pages/whats-happening/work-in-progress/index.mdx:49]:
```suggestion
    title="IBM Grid template"
```

**Review by jillianhowarth — CHANGES_REQUESTED** [review]
@mariuszmickiewicz just a couple content tweaks to make the cards more clear, but otherwise it looks great! Let me know if you have any questions 🙂

**Comment by mariuszmickiewicz** [comment]
@jillianhowarth Thanks for the updates!

**Comment by mariuszmickiewicz** [comment]
@jillianhowarth Do you know who could help with fathom codes?

**Comment by joshblack** [comment]
Just pushed up a change with the goals, hope that works @jillianhowarth @mariuszmickiewicz!

**Review by mariuszmickiewicz — COMMENTED** [review]
I can't approve this update somehow (as I'm the pull requester), but if anyone can that would be cool.

**Review by jillianhowarth — APPROVED** [review]
🎉

**Comment by mariuszmickiewicz** [comment]
Hmm, this doesn't seem to be visible on the live site.

**Comment by janchild** [comment]
You're right, @mariuszmickiewicz. 

@alisonjoseph Do you mind checking to see why these WIP page additions aren't showing on the site?

**Comment by tw15egan** [comment]
Seems like they were accidentally removed in this PR: https://github.com/carbon-design-system/carbon-website/pull/1827/files

I'll get up a PR to fix this, @mariuszmickiewicz!

**Pull request merged**